### DISCUSSION TITLE
Compatibility between the firmware and Ender-3 Pro + Sprite Pro kit

### DISCUSSION THREAD

**Discussion by karolane-lc** [mriscoc/Ender3V2S1#182] [Q&A] [answered]
Hi!
Will this firmware works on an Ender-3 Pro equiped with the Sprite pro kit extruder and the CR-touch ?

**Comment by mriscoc** [comment, marked as answer]
This firmware requires a DWIN/DACAI screen unit.

**Reply by karolane-lc** [reply]
oh I see ! well thanks for fast answering ! :(
